web/src/tippyjs.js

```diff
@@ -501,6 +501,23 @@ export function initialize() {
             defaultMessage: "View user card (u)",
         }),
         delay: LONG_HOVER_DELAY,
+        onShow(instance) {
+            if (!document.body.contains(instance.reference)) {
+                return false;
+            }
```

Comment:
Seems like only this is needed to fix your reproducer.

Reply:
I suggested this approach; however, you advised using the `hide_tooltip_if_reference_removed` function. Do you want me to refactor this?  [Message Link](https://chat.zulip.org/#narrow/stream/6-frontend/topic/tippy.20sticking.20around/near/1543052)